content/tutorials/getting-started/index.md

```diff
@@ -173,53 +182,41 @@ Gatling Enterprise requires a package built from a Gatling project in order to r
 To create a package, run the `enterprisePackage.bat` (if Windows) or `enterprisePackage.sh` (if Linux/MacOS) file, Gatling will start creating your first package:
 
 ```
-$ ./bin/enterprisePackage.sh
-GATLING_HOME is set to .../gatling-charts-highcharts-bundle-{{< var gatlingVersion >}}
```

Comment:
package part is useless here (only useful for self hosted)
I think we should only mention the **[2] Run the Simulation on Gatling Enterprise Cloud** part, and completely rework the step 3